services/backup/backup_service.go

```diff
@@ -142,6 +185,26 @@ func (s *Service) PerformBackup(ctx context.Context, params PerformBackupParams)
 	return artifact.ID, nil
 }
 
+func checkMongoBackupPreconditions(q *reform.Querier, service *models.Service, scheduleID string) error {
+	tasks, err := models.FindScheduledTasks(q, models.ScheduledTasksFilter{
+		Disabled:  pointer.ToBool(false),
+		ServiceID: service.ServiceID,
+		Mode:      models.PITR,
+	})
+	if err != nil {
+		return err
+	}
+
+	for _, task := range tasks {
+		if task.ID != scheduleID {
+			return status.Errorf(codes.FailedPrecondition, "Can't perform backup because service %s has configured scheduled "+
```

Comment:
I suggest to turn it to this:
"Can't make a backup because service %s already contains scheduled PITR backups. Please disable them if you want to make another backup."

Reply:
Done partially, a left `has` instead of `contains`. IMHO this word fits better.